teflo_datarouter_plugin/helpers.py

```diff
@@ -35,15 +35,30 @@ def validate_compose_payload_content(tar_dir_list):
 
 
 def validate_struc_before_compose(payload_dir):
+    """ Validating payload contain results/ dir and is not empty."""
+
     dir_contenct = []
     for dirname, dirnames, filenames in os.walk(payload_dir):
         for subdirname in dirnames:
             dir_contenct.append(os.path.join(dirname, subdirname))
 
         for filename in filenames:
             dir_contenct.append(os.path.join(dirname, filename))
+    substring_in_list = any('/results/' in s for s in dir_contenct)
+    strings_with_substring = [s for s in dir_contenct if '/results/' in s]
+    if substring_in_list and strings_with_substring:
+        return True
+    else:
+        raise TefloReportError("Payload Dir structure is incorrect!")
+
+
+def send_get_req(dr_token_url, body):
```

Comment:
the method name is send_get_request but it is actually doing a post. what isthis method needed for , provide a description